common/arg.cpp

```diff
@@ -813,13 +813,18 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
     ).set_env("LLAMA_ARG_FLASH_ATTN"));
     add_opt(common_arg(
         {"-p", "--prompt"}, "PROMPT",
-        ex == LLAMA_EXAMPLE_MAIN
-            ? "prompt to start generation with\nif -cnv is set, this will be used as system prompt"
-            : "prompt to start generation with",
+        "prompt to start generation with",
```

Comment:
Add a small clarification, so users already using will not be confused

```suggestion
        "prompt to start generation with; for system message, use -sys",
```

Btw, how is this `-p` now being used in `-cnv` mode? (Would be nice to have a small clarification on that too)

Reply:
Would it make sense to pre-start a conversation if you use -p?

Reply:
Hmmm, not so simple though as it needs to be stacked with system prompt. How about just outputting a warning that -p is ignored in this case?

Reply:
Yes a warning would be better

Reply:
> Would it make sense to pre-start a conversation if you use -p?

This seems to be suitable with the interactive (but with non-interactive-**first**), but sounds like too much work for the moment.